docs-conceptual/azps-2.0.0/migrate-az-1.0.0.md

```diff
@@ -42,146 +47,181 @@ This document describes the changes between the 6.x versions of AzureRM and Az v
 
 ### Windows PowerShell 5.1 and .NET 4.7.2
-- Using Az with Windows PowerShell 5.1 requires the installation of .NET 4.7.2. However, using Az with PowerShell Core does not require .NET 4.7.2. 
+
+Using Az with Windows PowerShell 5.1 requires the installation of .NET 4.7.2. However, using Az with PowerShell Core does not require .NET 4.7.2. 
 
 ### Temporary removal of User login using PSCredential
-- Due to changes in the authentication flow for .NET Standard, we are temporarily removing user login via PSCredential. This capability will be re-introduced in the 1/15/2019 release for Windows PowerShell 5.1. This is discussed in detail in [this issue.](https://github.com/Azure/azure-powershell/issues/7430)
 
-### Default Device Code login instead of Web Browser prompt
-- Due to changes in the authentication flow for .NET Standard, we are using device login as the default login flow during interactive login. Web browser based login will be re-introduced for Windows PowerShell 5.1 as the default in the 1/15/2019 release. At that time, users will be able to choose device login using a Switch parameter.
+Due to changes in the authentication flow for .NET Standard, we are temporarily removing user login via PSCredential. This capability will be re-introduced in the 1/15/2019 release for Windows PowerShell 5.1. This is discussed in detail in [this GitHub issue.](https://github.com/Azure/azure-powershell/issues/7430)
+
+### Default device code login instead of web browser prompt
+
+Due to changes in the authentication flow for .NET Standard, we are using device login as the default login flow during interactive login. Web browser based login will be re-introduced for Windows PowerShell 5.1 as the default in the 1/15/2019 release. At that time, users will be able to choose device login using a Switch parameter.
 
 ## Module breaking changes
 
+This section details specific breaking changes for individual modules and cmdlets.
+
 ### Az.ApiManagement (previously AzureRM.ApiManagement)
-- Removing the following cmdlets:
+
+- Removed the following cmdlets:
   - New-AzureRmApiManagementHostnameConfiguration
   - Set-AzureRmApiManagementHostnames
   - Update-AzureRmApiManagementDeployment
   - Import-AzureRmApiManagementHostnameCertificate
   - Use **Set-AzApiManagement** cmdlet to set these properites instead
-- Following properties were removed
+- Removed the following properties:
   - Removed property `PortalHostnameConfiguration`, `ProxyHostnameConfiguration`, `ManagementHostnameConfiguration` and `ScmHostnameConfiguration` of type `PsApiManagementHostnameConfiguration` from `PsApiManagementContext`. Instead use `PortalCustomHostnameConfiguration`, `ProxyCustomHostnameConfiguration`, `ManagementCustomHostnameConfiguration` and `ScmCustomHostnameConfiguration` of type `PsApiManagementCustomHostNameConfiguration`.
```

Comment:
The style guide we have for PowerShell says that properties, class names, type names, and cmdlet parameters should be in bold and cmdlet names should be in backticks (code fences).

See https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/contributing/3-STYLE-GUIDE.md#formatting-syntax-elements

Reply:
I've taken a general task to fix the AzPS docs to conform to this style guide and will update the entire docset at one time. I'm assuming that the reference already follows these guidelines?

Reply:
Read the whole contributor guide in that folder. There are many aspects for how we format the PowerShell docs. Let me know if you have any questions. It is a living document. We add or change things as necessary. But this has help drive consistency in my docset.